examples/minimal.rs

```diff
@@ -1,6 +1,9 @@
+extern crate log;
```

Comment:
Wouldn't it be better to publically extern it, so that it's known what version wlroots-rs is using when applications link to it? It would tie them to the version that wlroots-rs supports, however they wouldn't be able to pass a different version anyways correct?

I am out of the loop  when it comes to this library so I'm unsure how best to expose it as a dependency.

Reply:
I've looked a little into how other libraries (including logging implementations) use it, and it doesn't seem to be common practice.

For the user this means to add wlroots-rs and the log crate into their Cargo.toml, and `use` the macros from `log`, after initializing the logger.